src/browser/getRenderFunc.js

```diff
@@ -1,4 +1,7 @@
 export default function getRenderFunc(variant) {
+  if (typeof variant === 'undefined') {
```

Comment:
you probably want == null here, in case it’s null

Reply:
Good call, I'll make that change.